boilerplate/app/screens/DemoShowroomScreen/DemoShowroomScreen.tsx

```diff
@@ -181,6 +182,12 @@ export const DemoShowroomScreen: FC<DemoTabScreenProps<"DemoShowroom">> =
             stickySectionHeadersEnabled={false}
             sections={Object.values(Demos)}
             renderItem={({ item }) => item}
+            renderScrollComponent={({ contentContainerStyle }) => (
+              <KeyboardAwareScrollView
+                contentContainerStyle={contentContainerStyle}
+                bottomOffset={50}
+              />
+            )}
```

Comment:
By adding this, we get normal keyboard avoid behavior within a `SectionList`. From here, we can do a couple of things: 

* Extract 50 to a default constant for bottom offset
* Create a new component (e.g. `SectionListWithKeyboardAwareScrollView` that brings this behavior by default
* Add more documentation overall in Ignite about keyboard avoid behavior for SectionList, FlatList FlashList, etc.

Reply:
Here's also a demo:


https://github.com/user-attachments/assets/48ec3cca-967b-4dd9-99de-3baaea676edb

Reply:
Do you know if we can use this on the `LoginScreen` as well to ensure the "Tap to sign in!" button stays visible? or is that maybe too complicated for now?

Reply:
@lindboe yes, it's possible. Check this video [here](https://github.com/infinitered/ignite/pull/2722#discussion_r1700460133).

Reply:
Oh I see, I missed that's what that video was doing, I saw the multiline input and thought it was still the demo screen 👍